lld/MachO/ObjC.cpp

```diff
@@ -320,3 +332,915 @@ void objc::checkCategories() {
       }
   }
 }
+
+namespace {
+
+class ObjcCategoryMerger {
+  // Information about an input category
+  struct InfoInputCategory {
+    ConcatInputSection *catBodyIsec;
```

Comment:
Can you switch the order of these fields following other code flow/data -- catListIsec followed by catBodyIsec?